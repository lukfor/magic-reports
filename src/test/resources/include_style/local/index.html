{{include_style("index.css")}}

<p class="example">example in red</p>