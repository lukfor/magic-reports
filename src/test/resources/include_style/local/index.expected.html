<!-- index.css -->
<link rel="stylesheet" href="include_style_local_files/index.css">

<p class="example">example in red</p>